src/web/app/pages-instructor/instructor-course-edit-page/instructor-edit-panel/instructor-edit-panel.component.html

```diff
@@ -72,15 +72,17 @@
         <input id="checkbox-display-instructor-{{ instructorIndex + 1 }}" checked=""
                type="checkbox" [ngModel]="instructor.isDisplayedToStudents"
                (ngModelChange)="triggerModelChange('isDisplayedToStudents', $event)" [disabled]="!instructor.isEditing">
+        &nbsp;
         <span ngbTooltip="If this is unselected, the instructor will be completely invisible to students. E.g. to give access to a colleague for ‘auditing’ your course">Display to students as:</span>
       </label>
       <div class="col-sm-9">
         <input id="displayed-name-instructor-{{ instructorIndex + 1 }}" class="form-control"
                type="text"
                (ngModelChange)="triggerModelChange('displayedToStudentsAs', $event)"
                [disabled]="!instructor.isDisplayedToStudents || !instructor.isEditing"
-               placeholder="E.g.Co-lecturer, Teaching Assistant"
-               [ngModel]="instructor.isDisplayedToStudents ? instructor.displayedToStudentsAs : '(This instructor will NOT be displayed to students)'">
+               [ngModel]="instructor.isDisplayedToStudents ? instructor.displayedToStudentsAs : '(This instructor will NOT be displayed to students)'"
+               [defaultValue]="'Instructor'"
```

Comment:
So we should delete this?

Reply:
Wait sorry I think we shouldn't. If the user types something then backspaces the model will be empty. The model should get populated with the default value here in this scenario.

Reply:
I think there is no need to do this. Backend will populate the value to `Instructor` anyway so when you click save, the input box will be populated.

Reply:
That was the original intent yes but @wkurniawan07 raised a valid point that the frontend shouldn't make assumptions about what happens in the backend https://github.com/TEAMMATES/teammates/pull/10497#discussion_r462234617